package.json

```diff
@@ -22,6 +23,7 @@
     "dev:https": "gatsby develop --https --host localhost.corp.adobe.com --port 9000",
     "build": "gatsby build",
     "serve": "gatsby serve",
+    "fix-markdown": "markdown-cleaner",
```

Comment:
Is this the original package you worked on for parliament?

https://github.com/adobe/parliament-markdown-cleaner

Reply:
Yeah, but it wasn't in this separate package at the time. See the full explanation in our migration channel. :)